chromium_src/chrome/browser/ui/views/tabs/tab_style_views.cc

```diff
@@ -18,170 +18,4 @@
-                                      bool force_active,
-                                      RenderUnits render_units) const {
-  if (!ShouldShowVerticalTabs())
-    return BraveGM2TabStyle::GetPath(path_type, scale, force_active,
-                                     render_units);
-
-  const int stroke_thickness = GetStrokeThickness();
-  gfx::RectF aligned_bounds =
-      ScaleAndAlignBounds(tab()->bounds(), scale, stroke_thickness);
-
-  // Calculate the bounds of the actual path.
-  float tab_top = aligned_bounds.y();
-  float tab_left = aligned_bounds.x();
-  float tab_right = aligned_bounds.right();
-  float tab_bottom = aligned_bounds.bottom();
-
-  const float stroke_adjustment = stroke_thickness * scale;
-  if (path_type == PathType::kInteriorClip) {
-    // Inside of the border runs |stroke_thickness| inside the outer edge.
-    tab_left += stroke_adjustment;
-    tab_right -= stroke_adjustment;
-    tab_top += stroke_adjustment;
-    tab_bottom -= stroke_adjustment;
-  } else if (path_type == PathType::kFill || path_type == PathType::kBorder) {
-    tab_left += 0.5f * stroke_adjustment;
-    tab_right -= 0.5f * stroke_adjustment;
-    tab_top += 0.5f * stroke_adjustment;
-    tab_bottom -= 0.5f * stroke_adjustment;
-  }
-
-  constexpr int kRadius = 8;
-  SkPath path;
-  path.addRoundRect({tab_left, tab_top, tab_right, tab_bottom}, kRadius * scale,
-                    kRadius * scale);
-
-  // Convert path to be relative to the tab origin.
-  gfx::PointF origin(tab()->origin());
-  origin.Scale(scale);
-  path.offset(-origin.x(), -origin.y());
-
-  // Possibly convert back to DIPs.
-  if (render_units == RenderUnits::kDips && scale != 1.0f)
-    path.transform(SkMatrix::Scale(1.0f / scale, 1.0f / scale));
-
-  return path;
-}
-
-TabStyle::SeparatorBounds BraveVerticalTabStyle::GetSeparatorBounds(
-    float scale) const {
-  if (ShouldShowVerticalTabs())
-    return {};
-
-  return BraveGM2TabStyle::GetSeparatorBounds(scale);
-}
-
-void BraveVerticalTabStyle::PaintTab(gfx::Canvas* canvas) const {
-  BraveGM2TabStyle::PaintTab(canvas);
-  if (!ShouldShowVerticalTabs()) {
-    return;
-  }
-
-  if (tab()->IsActive() || IsHoverActive() || tab()->data().pinned) {
-    const auto* widget = tab()->GetWidget();
-    DCHECK(widget);
-    const SkColor tab_stroke_color =
-        widget->GetColorProvider()->GetColor(kColorBraveVerticalTabSeparator);
-    PaintBackgroundStroke(canvas, TabActive::kActive, tab_stroke_color);
-  }
-}
-
-bool BraveVerticalTabStyle::ShouldShowVerticalTabs() const {
-  return tabs::utils::ShouldShowVerticalTabs(tab()->controller()->GetBrowser());
-}
-
-}  // namespace
-
-std::unique_ptr<TabStyleViews> TabStyleViews::CreateForTab(Tab* tab) {
-  if (base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs))
-    return std::make_unique<BraveVerticalTabStyle>(tab);
-
-  return std::make_unique<BraveGM2TabStyle>(tab);
-}
+#include "brave/browser/ui/views/tabs/brave_tab_style_views.inc"
```

Comment:
Ugh, Both lint and format doesn't seem to work. Let me try fixing them first.

Reply:
@simonhong @goodov I've checking out linter and git cl format. We can configure linter so that it can run against .inc files. But cl format is not that configurable, so there wasn't no much choice. So instead of fixing them I tried renaming `.inc` to `.inc.cc` and removed it from BUILD.gn - so that our compiler doesn't try to compile it out of the embedder file.

And it seemed to work and I checked these things
- Changes in `.inc.cc` file is detected even when we remove it from BUILD.gn? -> `Yes`
- Linter runs on it? -> `Yes`
- Formatter runs on it? -> `Yes`

Do you think it's okay splitting the file like this? I'm not sure what could be the side effects of not listing `inc.cc` file in BUILD.gn.

Reply:
If it's not listed in BUILD.gn, maybe modifying that file doesn't trigger rebuild?

Reply:
> Changes in .inc.cc file is detected even when we remove it from BUILD.gn? -> Yes

> Oh, it's interesting. touching inc.cc file triggers rebuild even it's not included in BUILD.gn!

Yeah, really interesting 😄 I thought it might not be tracked but it's triggered.